packages/actor-rdf-dereference-file/components/Actor/RdfDereference/File.jsonld

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/contexts/comunica-actor-rdf-dereference-file.jsonld",
+    "https://linkedsoftwaredependencies.org/contexts/comunica-bus-rdf-dereference.jsonld"
+  ],
+  "@id": "npmd:@comunica/actor-rdf-dereference-file",
+  "components": [
+    {
+      "@id": "cardf:Actor/RdfDereference/File",
+      "@type": "Class",
+      "extends": "cbrd:Actor/RdfDereference",
+      "requireElement": "ActorRdfDereferenceFile",
+      "comment": "A comunica File RDF Dereference Actor.",
+      "parameters": [
+        {
+          "@id": "cardf:mediatorRdfParse",
+          "comment": "The RDF parse mediator",
+          "required": true,
+          "unique": true
+        },
+        {
+          "@id": "cardf:mediaMappingEntry",
+          "comment": "Maps an extension to a file type",
+          "required": true,
+          "default": [
+            { "cardf:extension": ".ttl",      "cardf:mediaType": "text/turtle" },
```

Comment:
Was there a particular reason to include the dots in the extension? If not, I'd remove them.

Reply:
Because `path.extname` returns the dot, so to be consistent with that. But I don't really care either way.

Reply:
They apparently do that for distinguishing between cases where there is an extension, and where there is none. (To avoid `undefined` I guess)

In any case, this is not a concern in our case, so we can remove the dot I guess. Saves us some bytes :p